mod.ts

```diff
@@ -390,6 +390,19 @@ export type EventOIDCJWTPreCreateHookResponse =
   | HookResponseDisallowed
   | Pick<HookResponseAllowed, "is_allowed" | "mutations">;
 
+export interface EventOIDCIDTokenPreCreate extends HookEventBase {
+  type: "oidc.id_token.pre_create";
+  payload: {
+    user: User;
+    identities: Identity[];
+    jwt: JWT;
```

Comment:
@louischan-oursky Should we rename this to `id_token`? As discussed before the name `jwt` is confusing.

Reply:
Addressed.